### PR TITLE
Entity Page - Download File

### PR DESCRIPTION
**Pull request by ciaranschutte**

### CODE REVIEW

File: src/components/EntityPage/File/index.js [hunk 1]
```diff
@@ -339,34 +340,37 @@ const FileEntity = ({ api, fileId }) => {
...
                 </EntityContentSection>
                 <EntityContentDivider />
-                <EntityContentSection title="Associated Participants/Biospecimens">
-                  <BaseDataTable
-                    loading={file.isLoading}
-                    data={particpantBiospecimenData(file.data)}
-                    columns={particpantBiospecimenColumns}
-                  />
-                </EntityContentSection>
+                <EntityContentSection title="Associated Participants/Biospecimens" />
```
Comment (nyanofthemoon) [inline src/components/EntityPage/File/index.js]: No content in these?

File: src/components/EntityPage/File/index.js [hunk 2]
```diff
@@ -39,17 +47,13 @@ const fileQuery = `query ($sqon: JSON) {
           controlled_access
           created_at
           data_type
-          experiment_strategies
```
Comment (nyanofthemoon) [inline src/components/EntityPage/File/index.js]: Fields don't exist anymore?

File: src/components/FileRepo/DownloadFileButton.js [hunk 3]
```diff
@@ -28,26 +27,32 @@ const downloadFile = async ({ kfId, api }) => {
...
+  withApi,
+)(
   ({
     kfId,
     theme,
     effects: { setToast },
     state: { integrationTokens },
     gen3Key = integrationTokens[GEN3],
     api,
+    render,
```
Comment (nyanofthemoon) [inline src/components/FileRepo/DownloadFileButton.js]: A bit conflicting with naming for React's `render` method, but I understand.

File: src/components/EntityPage/File/index.js (hunk 2 above)
Comment (ciaranschutte) [reply src/components/EntityPage/File/index.js]: Nope. data issue. following up on this.

File: src/components/EntityPage/File/index.js (hunk 1 above)
Comment (ciaranschutte) [reply src/components/EntityPage/File/index.js]: had to remove as workaround for data issue, will be added back when merging

File: src/components/FileRepo/DownloadFileButton.js (hunk 3 above)
Comment (hlminh2000) [reply src/components/FileRepo/DownloadFileButton.js]: react's own doc of the pattern actually uses "render" for its example :P

**Comment by ciaranschutte** [comment]
@hlminh2000 sounds good!

**Pull request merged**